spec/lib/brexit_checker/results_audiences_spec.rb

```diff
@@ -33,19 +92,23 @@
 
     context "actions and criteria are provided" do
       let(:result) { described_class.populate_business_groups(actions, selected_criteria) }
+
       it "produces an hash of actions and criteria" do
-        expect(result).to eq(
-          actions: actions,
-          criteria: %w(owns-operates-business-organisation aero-space forestry),
-        )
+        expect(result[:actions]).to eq(actions)
+        expect(result[:criteria].map(&:key)).to match_array([criteria1.key, criteria2.key, criteria3.key])
       end
 
       it "excludes answer criteria that are not relevant to actions" do
         expect(result[:criteria].length).to eq(3)
-        expect(result[:criteria]).not_to include("exports-enchanted-goods")
-        expect(result[:criteria]).to include("owns-operates-business-organisation")
-        expect(result[:criteria]).to include("aero-space")
-        expect(result[:criteria]).to include("forestry")
+        expect(result[:criteria].map(&:key)).not_to include("exports-enchanted-goods")
```

Comment:
It might be nice to assign `result[:criteria].map(&:key)` to a variable and use that in the expect statements to make it a bit more readable.

Reply:
good suggestion, done